collab-folder/src/core/relation.rs

```diff
@@ -126,6 +180,10 @@ impl ChildrenArray {
       .and_then(view_identifier_from_value)
   }
 
+  pub fn insert_child_with_txn(&self, txn: &mut TransactionMut, index: u32, child: ViewIdentifier) {
```

Comment:
It seems like a passthrough func, why not use self.0.insert(txn, index, child); directly in the caller.

Reply:
I think this method would be better as a standalone function, as it appears to be a common and frequently used operation.